packages/react-core/src/components/Menu/MenuContent.tsx

```diff
@@ -36,7 +36,7 @@ export const MenuContent = React.forwardRef((props: MenuContentProps, ref: React
           style={
             {
               '--pf-c-menu__content--Height': menuHeight || 'auto',
-              '--pf-c-menu__content--MaxHeight': !isScrollable ? maxMenuHeight || 'auto' : ''
+              '--pf-c-menu__content--MaxHeight': maxMenuHeight || (!isScrollable ? 'auto' : '')
```

Comment:
these vars are already defined as `auto` in the CSS - I would only set them if `menuHeight` and/or `maxMenuHeight` are defined. Can you think of a reason they always need to be set inline?

https://github.com/patternfly/patternfly/blob/a67b16945bec19cae3916d9688fe1e9c6cba2576/src/patternfly/components/Menu/menu.scss#L31-L32

Reply:
I can choose only to set them if the consumer passes a value - easy peasy

Reply:
I was not sure if there was a reason that originally it only sets 'auto' if it's not scrollable

Reply:
@nicolethoen I can't think of why it would be necessary, it's redundant with the component's CSS. My guess is it's a miscommunication from the original issue - https://github.com/patternfly/patternfly-react/issues/6367

> Adds a .pf-m-scrollable variant to the menu that replaces previously having to set --pf-c-menu__content--MaxHeight var to make the menu scrollable. It gives the content height a default max-height. You can still set the max-height using that var if you want something other than the default.

From the description, it kind of sounds like it's saying you would either use the scrollable class or set the var if you don't want the default that the scrollable class provides.